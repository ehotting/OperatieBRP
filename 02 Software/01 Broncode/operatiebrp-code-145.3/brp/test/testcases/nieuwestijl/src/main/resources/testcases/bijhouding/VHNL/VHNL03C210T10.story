Meta:
@status                 Klaar
@regels                 R2314
@usecase                UCS-BY.HG

Narrative: R2413 Een toelichting ontlening is verplicht bij het deblokkeren van een melding

Scenario: Ontleningstoelichting niet opgegeven en niet meegestuurd
            LT : VHNL03C210T10

Given alle personen zijn verwijderd

Given enkel initiele vulling uit bestand /LO3PL/VHNL03C210-001.xls
Given enkel initiele vulling uit bestand /LO3PL/VHNL03C210-002.xls

When voer een bijhouding uit VHNL03C210T10.xml namens partij 'Gemeente BRP 1'

Then heeft het antwoordbericht verwerking Foutief
Then is het antwoordbericht gelijk aan /testcases/bijhouding/VHNL/expected/VHNL03C210T10.xml voor expressie //brp:bhg_hgpRegistreerHuwelijkGeregistreerdPartnerschap_R

Then is in de database de persoon met bsn 715347913 niet als PARTNER betrokken bij een HUWELIJK
Then is in de database de persoon met bsn 531122025 niet als PARTNER betrokken bij een HUWELIJK

