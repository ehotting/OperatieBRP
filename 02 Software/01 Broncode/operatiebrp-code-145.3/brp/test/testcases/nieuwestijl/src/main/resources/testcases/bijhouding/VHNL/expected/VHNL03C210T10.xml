<brp:bhg_hgpRegistreerHuwelijkGeregistreerdPartnerschap_R xmlns:brp="http://www.bzk.nl/brp/brp0200">
   <brp:stuurgegevens>
      <brp:zendendePartij>199901</brp:zendendePartij>
      <brp:zendendeSysteem>BRP</brp:zendendeSysteem>
      <brp:referentienummer>*</brp:referentienummer>
      <brp:crossReferentienummer>00000000-0000-0000-0042-200000000104</brp:crossReferentienummer>
      <brp:tijdstipVerzending>*</brp:tijdstipVerzending>
   </brp:stuurgegevens>
   <brp:resultaat>
      <brp:verwerking>Foutief</brp:verwerking>
      <brp:hoogsteMeldingsniveau>Fout</brp:hoogsteMeldingsniveau>
   </brp:resultaat>
   <brp:meldingen>
      <brp:melding brp:objecttype="Melding" brp:referentieID="comid.voltrekkingHuwelijkInNederland">
         <brp:regelCode>R2413</brp:regelCode>
         <brp:soortNaam>Fout</brp:soortNaam>
         <brp:melding>*</brp:melding>
      </brp:melding>
   </brp:meldingen>
   <brp:voltrekkingHuwelijkInNederland brp:objecttype="AdministratieveHandeling">
      <brp:partijCode>507013</brp:partijCode>
      <brp:tijdstipRegistratie>*</brp:tijdstipRegistratie>
      <brp:gedeblokkeerdeMeldingen>
         <brp:gedeblokkeerdeMelding brp:objecttype="GedeblokkeerdeMelding" brp:referentieID="comid.voltrekkingHuwelijkInNederland.melding1">
            <brp:regelCode>R1867</brp:regelCode>
            <brp:melding>Een persoon die als partner bij een te registreren huwelijk of geregistreerd partnerschap is betrokken staat onder curatele.</brp:melding>
         </brp:gedeblokkeerdeMelding>
      </brp:gedeblokkeerdeMeldingen>
   </brp:voltrekkingHuwelijkInNederland>
</brp:bhg_hgpRegistreerHuwelijkGeregistreerdPartnerschap_R>
